packages/os/healthdog-toml

```diff
@@ -0,0 +1,7 @@
+metrics_url = "{{settings.healthdog.metrics-url}}"
+send_metrics = {{settings.healthdog.send-metrics}}
+service_health = [{{#each settings.healthdog.service-health as |service| ~}}"{{service}}",{{/each~}}]
```

Comment:
What is this doing?  I don't think the syntax is obvious and I can't find documentation on it.  Would it be clearer to use something like the `join_map` we have today, but for a list?

Reply:
This is, indeed, doing a join to surround each member of the vector of strings with quotes and a trailing comma. Is `join_map` a feature of the Handlebars crate? Or are you saying there is an example of how to handle a vector that exists in our code? (in Schnauzer?)

Reply:
Yeah, schnauzer has a helpers module with things like `join_map`, `default`, etc.

Reply:
It doesn't look like there is an existing helper in schnauzer for this, but the handlebars crate comes out-of-the-box with the ability to write this as I have here.

Reply:
I think we could make a simple helper that was much clearer than that handlebars syntax, though...

Reply:
Created the helper in [f515d38](https://github.com/bottlerocket-os/bottlerocket/compare/d62f75d5060aa2f3ac38e7a7f0a7d25d29811b62..f515d383c9e181c8a531a02414b320ff9231a81f)